core-web/libs/ui/src/lib/components/dot-content-thumbnail/dot-content-thumbnail.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="thumbnail" [ngSwitch]="thumbnailType">
```

Comment:
This should be the `ng-container` so you can use the `ngSwitch` and then the :host is the only parent.

Reply:
Done!

Reply:
Done!